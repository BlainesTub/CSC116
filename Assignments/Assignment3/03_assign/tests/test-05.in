5
0 1 1
0 2 2
1 2 2.5
2 3 24
2 4 67
3 4 100
