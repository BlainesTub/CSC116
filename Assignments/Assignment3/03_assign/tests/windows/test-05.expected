Input matrix. Size : 5 x 5
 -  :   1 :   2 :  -  :  -  : 
  1 :  -  : 2.5 :  -  :  -  : 
  2 : 2.5 :  -  :  24 :  67 : 
 -  :  -  :  24 :  -  : 100 : 
 -  :  -  :  67 : 100 :  -  : 

Input Adjacency list. Size : 5
From City: A -> (B,1) (C,2) 
From City: B -> (A,1) (C,2.5) 
From City: C -> (A,2) (B,2.5) (D,24) (E,67) 
From City: D -> (C,24) (E,100) 
From City: E -> (C,67) (D,100) 

Minimum Spanning Tree. Size : 5
From City: A -> (B,1) (C,2) 
From City: B -> (A,1) 
From City: C -> (A,2) (D,24) (E,67) 
From City: D -> (C,24) 
From City: E -> (C,67) 

Cost of original matrix: 196.5
Cost of minimum spanning tree: 94
